packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -720,7 +746,7 @@
         padding-bottom: 1.6rem;
 
         @include mobile-or-tablet-screen {
-            min-height: unset;
+            // min-height: unset;
```

Comment:
Can you please help to remove this. 
Thank you

Reply:
done 👍